src/schematics/app/files/__directory__/__name__/BUILD.bazel__tmpl__

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+load("@build_bazel_rules_angular//:defs.bzl", "ng_module")
+load("@build_bazel_rules_nrwl//:webpack.bzl", "webpack_bundle")
+
+ng_module(
+  name = "compile",
+  srcs = glob(["**/*.ts"], exclude = ["e2e/**/*.ts"]),
+  deps = [],
+  tsconfig = "//:tsconfig.json"
+)
+
+genrule(
```

Comment:
this will change a lot of course - how do we prevent ppl creating a bunch of BUILD files like this? maybe a big warning somewhere? maybe a comment emitted here?